module-2/backend/week_one.md

```diff
@@ -1,32 +1,83 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+> get, displays view
+  redirect, calls for different uri
+  post, create new data/object
+  put, update data/object
+  delete, delete data/object

```

Comment:
We were thinking get, post, put, patch, delete. Don't believe redirect is an HTTP method/verb, though there are others.